src/internal/api/packages/deploy.go

```diff
@@ -24,8 +24,7 @@ import (
 
 // DeployPackage deploys a package to the Zarf cluster.
 func DeployPackage(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
// Deploy deploys a package to the Zarf cluster.
func Deploy(w http.ResponseWriter, r *http.Request) {
```

Since this is the packages package should we removed Packages here? and update [‎src/internal/api/start.go](https://github.com/defenseunicorns/zarf/blob/f2fc5513580c67dc441b69b06c283194466840d2/src/internal/api/start.go#L91) I know it is different than package but just wondering your opinion